taxi_service/urls.py

```diff
@@ -22,4 +23,5 @@
 urlpatterns = [
     path("admin/", admin.site.urls),
     path("", include("taxi.urls", namespace="taxi")),
+    path("accounts/", include("django.contrib.auth.urls")),
 ] + static(settings.STATIC_URL, document_root=settings.STATIC_ROOT)
```

Comment:
Serving static files using `static()` is appropriate for development. Remember to configure a proper static files server for production environments.